release-notes.txt

```diff
@@ -1,3 +1,536 @@
+
+Networking
+----------
+
++ gcoap:  add coaps forward proxy (#20454)
++ gnrc, nanocoap: add optional work-arounds for buggy CoAP servers (#20564)
++ nanocoap_fs: add nanoCoAP as VFS backend (remote CoAP fs) (#20687)
++ nanocoap_sock: add nanocoap_get_blockwise_to_buf() (#20690)
++ nanocoap_sock: implement separate response (#20266)
+* doc: fix links to GNRC master thesis (#20785)
+* driver/w5500: driver for the W5500 ethernet chip (#20301)
+* examples/gcoap: take full URI as input (#20554)
+* gnrc/ipv6: Check for overflow (#20771)
+* l2filter: mark module and related netopt options as experimental (#20634)
+* pkg/lwip: use sys/event for handling ISR and bhp (#18359)
+
+Packages
+--------
+
++ pkg/tflite-micro: add proper options to flatc (#20703)
++ pkg/wakaama: implement IPSO sensor objects in LwM2M (#20652)
++ pkg/wamr: Added support for THUMB_VFP in wamr Makefile (#20628)
+* Makefile.include: creation of CACHEDIR.TAG as a dependency of pkg-
+  prepare (#20689)
+* pkg/littlefs2: bump to v2.9.2 (#20591)
+* pkg/tflite-micro: Update tflite-micro to latest version. requires
+  update of pkg/flatbuffers as well. (#20683)
+* pkg/wamr: updated package to WAMR 1.3.3 (#20691)
+
+Boards
+------
+
++ boards/arduino-nano-33-ble-sense: add support for arduino-
+  nano-33-ble-sense (#20668)
++ boards/b-l072z-lrwan1: add missing adc feature (#20733)
++ boards/nucleo-f303ze: add pinout (#20717)
++ boards/nucleo-l452re: add ADC (#20778)
++ boards/sodaq-sara-sff: add status pin for SARA (#20279)
++ cpu/stm32: add ADC support for WB55 (#20773)
++ drivers/led: add LED_NUMOF and convenience inline functions (#20783)
++ drivers/ws281x: Add saul support (#20562)
+* boards/common/stm32: simplify ifdef logic (#20617)
+* boards/nRF52xx: correctly set JLINK_DEVICE (#20599)
+* boards/nucleo-l476rg & stm32l476g-disco: fix MCU table (#20751)
+* boards/stm32l0538-disco: enable cpy2remed (#20766)
+* boards/stm32l476g-disco: enable cpy2remed (#20734)
+* build system: clean up netif features (#20700)
+* cpu/{gd32v,stm32}/periph/adc: make ADC clock setable (#19630)
+
+CPU
+---
+
++ cpu/cortexm: add stack limit support for Cortex-M33 (#20633)
+* cpu/cortexm: rework bkpt instruction call on panic (#20616)
+* cpu/cortexm_common: print last active thread on stack corruption (#20709)
+* cpu/esp: Handle format print errors (#20596)
+* cpu/msp430: implement power management (#20613)
+* cpu/nrf5x: implement pm_off() for nRF53/9160 (#20590)
+* cpu/nrf5x_common: fix ztimer issue on warm-boot (#20665)
+* cpu/nrf5x_common: properly calibrate RC-based low-frequency clock (#20669)
+* cpu/sam0: Remove deprecated muxpos (#20525)
+* cpu/sam0_common: avoid bitfield usage (#20747)
+* cpu/samd21: avoid the use of bitfield (#20713)
+* cpu/samd5x: avoid the use of bitfields (#20714)
+* cpu/samd5x: handle CAN errors (#20667)
+* cpu/saml1x: avoid the use of bitfield in register calls (#20710)
+* cpu/saml21: avoid the use of bitfield in register call (#20708)
```

Comment:
This is IMO worth a note: The first steps towards updating the vendor header files and allowing the port of new SAM* hardware has been made by @dylad.